Module.php

```diff
@@ -176,6 +176,9 @@ protected function initPanels()
             $config['module'] = $this;
             $config['id'] = $id;
             $this->panels[$id] = Yii::createObject($config);
+            if (!$this->panels[$id]->isEnabled()) {
```

Comment:
That would break compatibility with custom panels. I'd prefer checking if method exists first.